hero/sez.json

```
@@ -0,0 +1,199 @@
+{
+    "name": "Sez",
+    "rarity": 5,
+    "classType": "thief",
+    "element": "ice",
+    "zodiac": "scorpio",
+    "specialtyChangeName": "",
+    "selfSkillBarName": "",
+    "background": [
+        "A cynical and quiet man, Sez Gaien is a living weapon known to many as the Berseker. After gaining power from an Acolyte to escape the homunculus experiments, he was branded as mass murderer and has been running ever since."
+    ],
+    "relations": [
+        { "hero": "aramintha", "relationType": "trust" },
+        { "hero": "ras", "relationType": "trust" },
+        { "hero": "kiris", "relationType": "grudge" }
+    ],
+    "stats": {
+        "base": {
+            "cp": 1328,
+            "atk": 99,
+            "hp": 322,
+            "spd": 113,
+            "def": 59,
+            "chc": 15,
+            "chd": 150,
+            "eff": 0,
+            "efr": 0,
+            "dac": 5
+        },
+        "max": {
+            "cp": 16762,
+            "atk": 1228,
+            "hp": 6266,
+            "spd": 113,
+            "def": 473,
+            "chc": 21,
+            "chd": 150,
+            "eff": 0,
+            "efr": 0,
+            "dac": 5
+        }
+    },
+    "skills": [
+        {
+            "isPassive": false,
+            "soulBurn": 0,
+            "selfSkillBarValue": 0,
+            "soulBurnEffect": "",
+            "awakenUpgrade": true,
+            "cooldown": 0,
+            "name": "Dark Shadow",
+            "soulAcquire": 1,
+            "description": "Penetrates the enemy with dark energy, making them unhealable for one turn. Damage dealt increases proportional to the amount of the enemy's lost Health.",
+            "enhancement": [
+                { "description": "+5% damage dealt", "resources": [{ "item": "gold", "qty": 4000 } , { "item": "molagora", "qty": 1 }] },
+                { "description": "+5% damage dealt", "resources": [{ "item": "gold", "qty": 8000 } , { "item": "molagora", "qty": 2}] },
+                { "description": "+5% damage dealt", "resources": [{ "item": "gold", "qty": 37000 } , { "item": "sharp spearhead", "qty": 5} , { "item": "molagora", "qty": 3 }] },
```

Items names cannot have spaces(or special characers). Spaces are replaced with '-' just like in file name as there is a (item).json file for each as well.
```suggestion
                { "description": "+5% damage dealt", "resources": [{ "item": "gold", "qty": 37000 } , { "item": "sharp-spearhead", "qty": 5} , { "item": "molagora", "qty": 3 }] },
```